[
  "test/test_app.py::AppViewTestCase::test_app_running", 
  "test/test_app.py::AppViewTestCase::test_authorized_access_after_login", 
  "test/test_app.py::AppViewTestCase::test_login_invalid_email", 
  "test/test_app.py::AppViewTestCase::test_login_invalid_password", 
  "test/test_app.py::AppViewTestCase::test_logout_user", 
  "test/test_app.py::AppViewTestCase::test_sign_up_data", 
  "test/test_app.py::AppViewTestCase::test_sign_up_page", 
  "test/test_app.py::AppViewTestCase::test_unauthorized_access_to_category", 
  "test/test_app.py::AppViewTestCase::test_unauthorized_access_to_dashboard", 
  "test/test_app.py::AppViewTestCase::test_unauthorized_user_access_recipe", 
  "test/test_app.py::AppViewTestCase::test_unauthorized_user_edit_recipe", 
  "test/test_user.py::TestUser::test_add_category", 
  "test/test_user.py::TestUser::test_application_running", 
  "test/test_user.py::TestUser::test_blank_category", 
  "test/test_user.py::TestUser::test_blank_email", 
  "test/test_user.py::TestUser::test_blank_names", 
  "test/test_user.py::TestUser::test_blank_password", 
  "test/test_user.py::TestUser::test_create_recipe_category", 
  "test/test_user.py::TestUser::test_dashboard", 
  "test/test_user.py::TestUser::test_dashboard_recipe_created_with_category", 
  "test/test_user.py::TestUser::test_dashboard_without_recipe", 
  "test/test_user.py::TestUser::test_delete_category", 
  "test/test_user.py::TestUser::test_delete_recipe_category", 
  "test/test_user.py::TestUser::test_duplicate_email", 
  "test/test_user.py::TestUser::test_edit_category", 
  "test/test_user.py::TestUser::test_edit_recipe_category", 
  "test/test_user.py::TestUser::test_empty_category_dashboard", 
  "test/test_user.py::TestUser::test_invalid_email", 
  "test/test_user.py::TestUser::test_invalid_password", 
  "test/test_user.py::TestUser::test_logging_running", 
  "test/test_user.py::TestUser::test_logout", 
  "test/test_user.py::TestUser::test_mismatch_signup_password", 
  "test/test_user.py::TestUser::test_successful_login", 
  "test/test_user.py::TestUser::test_user_registration_successful"
]